common/src/warp_runner/ui_adapter/mod.rs

```diff
@@ -38,49 +38,38 @@ use super::{
 pub struct Message {
     pub inner: warp::raygun::Message,
     pub in_reply_to: Option<(String, Vec<File>, DID)>,
-    pub lines_to_render: Option<String>,
-    pub is_mention: bool,
+    is_mention: i8,
```

Comment:
What does this represent overall?

Reply:
if the message is mentioning the current user. its used to decide whether to play a notification sound and highlight the message

Reply:
Maybe this should be changed to an enum to provide a clear representation

Reply:
maybe an option<bool> then?